Index: ucx-1.10.0~rc1/src/ucs/configure.m4
===================================================================
--- ucx-1.10.0~rc1.orig/src/ucs/configure.m4
+++ ucx-1.10.0~rc1/src/ucs/configure.m4
@@ -37,7 +37,7 @@ AS_IF([test "x$enable_backtrace_detail"
 	[
 	BT=1
 	AC_CHECK_HEADER([bfd.h], [], [AC_MSG_WARN([binutils headers not found])]; BT=0)
-	AC_CHECK_LIB(bfd, bfd_openr,  LIBS="$LIBS -lbfd", [AC_MSG_WARN([bfd library not found])];BT=0)
+	AC_CHECK_LIB(bfd, bfd_openr,  LIBS="$LIBS -Wl\,-Bstatic -lbfd -Wl\,-Bdynamic ", [AC_MSG_WARN([bfd library not found])];BT=0)
 	AC_CHECK_LIB(dl, dlopen, LIBS="$LIBS -ldl", [AC_MSG_WARN([dl library not found])];BT=0)
 	AC_CHECK_LIB(intl, main, LIBS="$LIBS -lintl", [AC_MSG_WARN([intl library not found])])
 	AC_CHECK_TYPES([struct dl_phdr_info], [], [AC_MSG_WARN([struct dl_phdr_info not defined])];BT=0,
