--- a/configure.ac
+++ b/configure.ac
@@ -353,17 +353,12 @@
                  ])
 AS_IF([test "x$with_docs_only" = xyes], [], [
 AC_CONFIG_LINKS([
-                 debian/compat:debian/compat
                  debian/copyright:debian/copyright
-                 debian/ucx.prerm:debian/ucx.prerm
                  ])
 AC_CONFIG_FILES([
                  ucx.spec
                  ucx.pc
                  contrib/rpmdef.sh
-                 debian/rules
-                 debian/control
-                 debian/changelog
                  src/ucp/Makefile
                  src/ucp/api/ucp_version.h
                  src/ucp/core/ucp_version.c
